Packs/CommonScripts/Scripts/ClearUserSession/README.md

```diff
@@ -0,0 +1,32 @@
+This script clears user sessions across multiple integrations for a list of usernames.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.10.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| user_name | List of names of the users to retrieve. |
+| brands | Which integrations brands to run the command for. If not provided, the command will run for all available integrations.  For multi-select provide a comma-separated list. For example: "Okta v2,Microsoft Graph User". |
+| verbose | Whether to retrieve human readable entry for every command or only the final result. True means to retrieve human readable entry for every command. False means to human readable only for the final result. |
+
+## Outputs
+
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| SessionClearingResults.UserName | The username for which the session clearing process was executed. | `string` |
+| SessionClearingResults.Entity | The username for which the session clearing process was executed.| `string`     |
+| SessionClearingResults.Result | The result of the session clearing process for the user (*Success* or *Failed*). | `string` |
+| SessionClearingResults.Brands | The integrations (e.g., *Microsoft Graph*, *Okta v2*) where the session clearing succeeded. | `string` |
+| SessionClearingResults.Message | Additional information or error details if the session clearing process failed. | `string` |
```

Comment:
This list of outputs does not match the list of outputs in the YML file nor the path. 
The path in the YML file is SessionClearResults
The path in the README file is SessionCleaingrResults
AND
The YML files lists: SessionClearResults.Source this does not appear in the README.
The README has:  SessionClearingResults.Entity and SessionClearingResults.Brands and these do not appear in the YML file.